source/magpie/group.d

```diff
@@ -718,6 +829,105 @@ public:
             return ret;
         }
     }
+
+    auto aggregate(int axis, Ops...)() @property
+    {
+        static if(axis)
```

Comment:
ditto, there is a lot of similar code and it makes it difficult to see what is different.

Reply:
I put all the initialisation into nested function :octocat: